ui/cryptomaterial/editor.go

```diff
@@ -161,45 +189,52 @@ func (e Editor) Layout(gtx layout.Context) layout.Dimensions {
 	}
 
 	return layout.UniformInset(e.m2).Layout(gtx, func(gtx C) D {
-		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-			layout.Rigid(func(gtx C) D {
-				return layout.Stack{}.Layout(gtx,
-					layout.Stacked(func(gtx C) D {
-						return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-							layout.Rigid(func(gtx C) D {
-								return e.editorLayout(gtx)
-							}),
-							layout.Rigid(func(gtx C) D {
-								if e.errorLabel.Text != "" {
-									inset := layout.Inset{
-										Top:  e.m2,
-										Left: e.m5,
-									}
-									return inset.Layout(gtx, func(gtx C) D {
-										return e.errorLabel.Layout(gtx)
-									})
+		return e.editorCard.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
+			return e.layout(gtx)
+		})
+
+	})
+}
+
+func (e Editor) layout(gtx C) D {
+	return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+		layout.Rigid(func(gtx C) D {
+			return layout.Stack{}.Layout(gtx,
+				layout.Stacked(func(gtx C) D {
+					return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							return e.editorLayout(gtx)
+						}),
+						layout.Rigid(func(gtx C) D {
+							if e.errorLabel.Text != "" {
+								inset := layout.Inset{
+									Top:  e.m2,
+									Left: e.m5,
 								}
-								return layout.Dimensions{}
-							}),
-						)
-					}),
-					layout.Stacked(func(gtx layout.Context) layout.Dimensions {
-						if e.IsTitleLabel {
-							return layout.Inset{
-								Top:  values.MarginPaddingMinus10,
-								Left: values.MarginPadding10,
-							}.Layout(gtx, func(gtx C) D {
-								return Card{Color: e.t.Color.Surface}.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
-									return e.TitleLabel.Layout(gtx)
+								return inset.Layout(gtx, func(gtx C) D {
+									return e.errorLabel.Layout(gtx)
 								})
+							}
+							return D{}
+						}),
+					)
+				}),
+				layout.Stacked(func(gtx layout.Context) D {
+					if e.IsTitleLabel {
+						return layout.Inset{
+							Top:  values.MarginPaddingMinus10,
+							Left: values.MarginPadding10,
+						}.Layout(gtx, func(gtx C) D {
+							return Card{Color: e.t.Color.Surface}.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
+								return e.TitleLabel.Layout(gtx)
 							})
-						}
-						return layout.Dimensions{}
-					}),
-				)
-			}),
-		)
-	})
+						})
+					}
+					return layout.Dimensions{}
```

Comment:
Instances of `layout.Dimensions` should use the shorthand `D`, same for `layout.Context{}`